pkg/client/migrations/1_v0.17.4.up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS cq_fetches
+(
+    cq_id                UUID NOT NULL,
+    cq_fetch_id          UUID NOT NULL,
```

Comment:
What's the difference between `cq_id` and `cq_fetch_id`? Also why the `cq_` prefix here?

Reply:
I agree we should remove the cq prefix here, as this table resides in its own schema